src/containers/App/partials/AppHeader.js

```diff
@@ -138,24 +153,23 @@ const AppHeader = React.memo(function AppHeader(props) {
       />
 
       <Toolbar>
-        <IconButton
-          onClick={onNavMenuToggle}
-          color="inherit" // Inherit color from `headerColor`.
-          edge="start"
-          size="small"
-          aria-haspopup="true"
-          aria-expanded={isNavMenuOpen}
-          aria-label={t(__translationGroup)`Toggle main menu`}
-        >
-          {isNavMenuOpen ? <CloseIcon /> : <MenuIcon />}
-        </IconButton>
-
```

Comment:
We need this toggle button to show the mobile navbar.

Reply:
Can we place the hamburger menu in the right corner instead? So the logo is not jumping around when the burger is appearing.